src/components/LoginPage.tsx

```
@@ -26,62 +30,86 @@ export function LoginPage({ onLogin }: LoginPageProps) {
+              </p>
             </div>
 
-            {error && (
-              <div className="alert alert-error mt-4">
-                <span>{error}</span>
+            <form onSubmit={handleSubmit}>
+              <div className="form-control">
+                <label className="label">
+                  <span className="label-text font-medium">GitHub Token</span>
+                </label>
+                <div className="relative">
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)